src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Calibrator;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Training;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<PlattCalibrator>), typeof(PlattCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", PlattCalibratorTransformer.LoadName)]
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<NaiveCalibrator>), typeof(NaiveCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", NaiveCalibratorTransformer.LoadName)]
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<PavCalibrator>), typeof(PavCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", PavCalibratorTransformer.LoadName)]
+
+namespace Microsoft.ML.Calibrator
+{
+
+    /// <summary>
+    /// An interface for probability calibrators.
+    /// </summary>
+    public interface ICalibrator
+    {
+        /// <summary> Given a classifier output, produce the probability </summary>
+        float PredictProbability(float output);
+
+        /// <summary> Get the summary of current calibrator settings </summary>
+        string GetSummary();
```

Comment:
I don't see any usages of this either in this repo, or internal one.
Shall we remove it?